http-client/Network/HTTP/Client.hs

```diff
@@ -1,5 +1,5 @@
 {-# LANGUAGE DeriveFunctor #-}
-{-# LANGUAGE DeriveTraversable #-}
+{-# LANGUAGE DeriveTraversable  #-}
```

Comment:
Yeah, sorry for that. I had my stylish-haskell on, it was late, I tried to fix it manually... I will do another PR to remove that.